bin/lint

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# This script looks for code offenses using linters
+# You can run it with `bin/lint`
+
+set -e
+
+echo "============================================"
+echo "== Looking for code offenses with Rubocop =="
+echo "============================================"
+bundle exec rubocop
```

Comment:
I think we should add reek here as well
In somewhere `brakeman` should also be executed 
You can also update the readme to use these scripts instead of using bundle exec ...

Reply:
- Good! I'll add reek to bin/lint and update the documentation with the new scripts
- About brakeman, it is running in bin/scan. I left an example in the PR description of how to run all the checks at the same time